utilities/transactions/write_unprepared_txn.cc

```diff
@@ -192,15 +206,20 @@ Status WriteUnpreparedTxn::FlushWriteBatchToDB(bool prepared) {
       db_impl_->immutable_db_options().two_write_queues);
   const bool DISABLE_MEMTABLE = true;
   uint64_t seq_used = kMaxSequenceNumber;
+  // We only set log_number_ if it hasn't been set, so that log_number_ always
```

Comment:
Who is "We"? where is the line that makes sure that log_number_ will not be reassigned?

Reply:
The logic is just in the WriteImpl call itself:

`log_number_ ? nullptr : &log_number_`

I can move that out to make things clearer.